src/components/Modal/Translate.vue

```diff
@@ -20,26 +20,44 @@
   -->
 
 <template>
-	<NcModal @close="$emit('close')">
+	<NcModal size="large" @close="$emit('close')">
 		<div class="translate-dialog">
 			<h2>{{ t('text', 'Translate') }}</h2>
-			<textarea v-model="input" />
-			<div class="language-selector">
-				<label for="fromLanguage">{{ t('text', 'Source language') }}</label>
-				<NcSelect v-model="fromLanguage"
-					input-id="fromLanguage"
-					:options="fromLanguages"
-					:append-to-body="false" />
+			<em>{{ t('text', 'You can translate parts of the document by selecting the text before using the translate function.') }}</em>
```

Comment:
The text should be "To translate individual parts of the text, select it before using the translate function." as mentioned by @jancborchardt [here](https://github.com/nextcloud/text/issues/3925#issuecomment-1566903096)